Makefile

```diff
@@ -181,7 +183,12 @@ test-examples-kustomize-against-HEAD: $(MYGOBIN)/kustomize $(MYGOBIN)/mdrip
 test-examples-kustomize-against-latest-release: $(MYGOBIN)/mdrip
 	./hack/testExamplesAgainstKustomize.sh v5@$(LATEST_RELEASE)
 
-
+# Pushes dependencies in the go.work file back to go.mod files of each workspace module.
+.PHONY: workspace-sync
+workspace sync:
```

Comment:
Do you have any specific reason to remove the hyphen here?
I think it is better to use a unified name here.

Reply:
No issue here, yes we can go with unified name